src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -807,3 +882,44 @@ def _get_exon_number(tx_exons: List, tx_pos: int) -> int:
                 break
             i += 1
         return i
+
+    @staticmethod
+    def _get_adjacent_exon(
+        tx_exons_genomic_coords: List[Tuple[int, int, int, int, int]],
+        strand: Strand,
+        start: Optional[int] = None,
+        end: Optional[int] = None,
+    ) -> int:
+        """Return the adjacent exon given a non-exonic breakpoint. For the positive strand, adjacent is defined
+        as the exon preceeding the breakpoint for the 5' end and the exon following the breakpoint for the 3' end.
+        For the negative strand, adjacent is defined as the exon following the breakpoint for the 5' end and the exon
+        preceeding the breakpoint for the 3' end.
+
+        :param: tx_exons_genomic_coords: List of tuples describing exons and genomic coordinates for a transcript. Each
+        tuple contains the transcript number (0-indexed), the transcript coordinates for the exon, and the genomic coordinates
+        for the exon.
+        :param: start: Genomic coordinate of brekapoint
+        :param: end: Genomic coordinate of breakpoint
+        :return: Exon number corresponding to adjacent exon. Will be 1-based
+        """
+        for i in range(len(tx_exons_genomic_coords) - 1):
+            exon = tx_exons_genomic_coords[i]
+            next_exon = tx_exons_genomic_coords[i + 1]
+            bp = start if start else end
+            if strand == strand.POSITIVE:
+                if bool(bp >= exon[4] and bp <= next_exon[3]):
+                    break
+            else:
+                if bool(bp >= next_exon[4] and bp <= exon[3]):
+                    break
```

Comment:
It's not necessary to cast `bool`. Lmk if this suggestion makes sense

```suggestion
            bp = start if start else end
            if strand == strand.POSITIVE:
                lte_exon = exon
                gte_exon = next_exon
            else:
                lte_exon = next_exon
                gte_exon = exon

            if bp >= lte_exon[4] and bp <= gte_exon[3]:
                break

```